src/test/java/com/faforever/client/patch/GameBinariesUpdateTaskTest.java

```diff
@@ -38,7 +41,7 @@
   public void setUp() throws Exception {
     MockitoAnnotations.initMocks(this);
 
-    instance = new GameBinariesUpdateTaskImpl(i18n, preferencesService, new ClientProperties());
+    instance = new GameBinariesUpdateTaskImpl(i18n, preferencesService, platformService, new ClientProperties());
```

Comment:
That should be something that you can test super easy just check if the correct call on the PlatformService was made

Reply:
I don't understand what the test would be.

Reply:
Sorry I mean in the ClientUpdateServiceImpl test

Reply:
What specifically do you want me to test?  That a file's permissions really got set the way we specified in the call?  Since that's the only thing this method does, aren't we running a unit test that Apache Commons would have already run themselves?

Reply:
Yes I want to to check if platformService.setUnixExecutableBit(binaryPath); was acalled